LICENSE

```diff
@@ -8,14 +8,14 @@ License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
 
 Parameters
 
-Licensor:             Babylonchain, Inc.
+Licensor:             Babylon Labs, Ltd.
 
 Licensed Work:        covenant-signer
-                      The Licensed Work is (c) 2024 Babylonchain, Inc.
+                      The Licensed Work is (c) 2023 Babylon Labs, Ltd.
```

Comment:
why 2023?

Reply:
tbh I have copied it from Babylon repo 😅

Reply:
ok merging it as is for now, we can fix date if necessary later